wp/plugin/wp.ml

```diff
@@ -18,149 +18,158 @@ module Typ = Extension.Type
 
 let name = "wp"
 
-let doc = "WP is a comparative analysis tool. It can compare two binaries \
-           and check that they behave in similar ways. It can also be used \
-           to check a single binary for certain behaviors."
+let doc = {|WP is a comparative analysis tool. It can compare two binaries
+            and check that they behave in similar ways. It can also be used
+            to check a single binary for certain behaviors.|}
 
 (* Mandatory arguments. *)
 
 let files = Cmd.arguments Typ.file
-    ~doc:"Path(s) to one or two binaries to analyze. If two binaries are \
-          specified, WP will run a comparative analysis."
+    ~doc:{|Path(s) to one or two binaries to analyze. If two binaries are
+           specified, WP will run a comparative analysis. If 0 or more than 2
+           files are give, WP will fail.|}
 
 let func = Cmd.parameter Typ.string ~aliases:["function"] "func"
-    ~doc:"Function to run the WP analysis on. If no function is specified or \
-          the function cannot be found in the binaries, the analysis will \
-          fail."
+    ~doc:{|Determines which function to verify. WP verifies a single function,
+           though calling it on the `main' function along with the `inline'
+           option will analyze the whole program. If no function is specified
+           or the function cannot be found in the binary/binaries, the analysis
+           will fail.|}
 
 (* Arguments that determine which properties CBAT should analyze. *)
 
 let precond = Cmd.parameter Typ.string "precond"
-    ~doc:"A custom precondition in SMT-LIB format to be used during the \
-          analysis. If no precondition is specified, a trivial precondition \
-          of `true' will be used."
+    ~doc:{|Allows the introduction of assertions to the beginning of a query.
```

Comment:
I'm not sure a user knows what a "query" is (I don't know what that is, for example). I'm also not sure the user will understand what it means to prune possible models (I think I do know what that one means though). 

I wonder if it would be simpler to say something like "Allows you to specify an assertion that WP will assume is true at the beginning of the function it is analyzing. Assertions are specified in the smt-lib2 format..."